src/app/clients/clients.component.html

```diff
@@ -2,15 +2,10 @@
   <div fxLayout="row" fxLayoutAlign="start center">
     <div class="search-box">
       <mat-form-field>
-        <input matInput placeholder="Filter by client name" (keyup)="applyFilter($event.target.value)" />
+        <input matInput placeholder="Filter by Client" (keyup)="applyFilter($event.target.value)" />
```

Comment:
Aren't these supposed to be localized?

Reply:
For now we are not but in the upstream, they have localized these, we will need to get those commits into our repo.